backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointResourceTest.java

```diff
@@ -358,6 +366,145 @@ void addCamelEndpoint() {
         }
     }
 
+    @Test
+    void addBogusCamelEndpoint() {
+
+        String tenant = "empty";
+        String userName = "user";
+        String identityHeaderValue = TestHelpers.encodeRHIdentityInfo(tenant, userName);
+        Header identityHeader = TestHelpers.createRHIdentityHeader(identityHeaderValue);
+
+        mockServerConfig.addMockRbacAccess(identityHeaderValue, MockServerClientConfig.RbacAccess.FULL_ACCESS);
+
+        CamelProperties cAttr = new CamelProperties();
+        cAttr.setDisableSslVerification(false);
+        cAttr.setUrl(String.format("https://%s", mockServerConfig.getRunningAddress()));
+        cAttr.setBasicAuthentication(new BasicAuthentication("testuser", "secret"));
+        Map<String, String> extras = new HashMap<>();
+        extras.put("template", "11");
+        cAttr.setExtras(extras);
+
+        Endpoint ep = new Endpoint();
+        ep.setType(EndpointType.CAMEL);
```

Comment:
Is it bogus because it doesn't have a subtype?

Reply:
subtypes are required for `EndpointType.Camel` - I wonder if we can enforce that now :thinking:

Reply:
Yes